src/server/ranch/RanchDirector.cpp

```diff
@@ -2399,23 +2404,31 @@ void RanchDirector::HandleHousingBuild(
     });
 
   auto housingUid = data::InvalidUid;
+  constexpr int16_t DoubleIncubatorId = 52;
+  constexpr int16_t SingleIncubatorId = 51;
```

Comment:
```suggestion
  static constexpr int16_t DoubleIncubatorId = 52;
  static constexpr int16_t SingleIncubatorId = 51;
```

Make them static so you don't have to capture them in lambdas. Also, if these IDs are re-used across the code, consider moving them to anonymous namespace at the beginning of the source file. This way, they can be re-used and refactored to use ItemRegistry later much easier. 

https://www.learncpp.com/cpp-tutorial/unnamed-and-inline-namespaces/

> **All content declared in an unnamed namespace is treated as if it is part of the parent namespace**. So even though function doSomething() is defined in the unnamed namespace, the function itself is accessible from the parent namespace (which in this case is the global namespace), which is why we can call doSomething() from main() without any qualifiers.

> This might make unnamed namespaces seem useless. **But the other effect of unnamed namespaces is that all identifiers inside an unnamed namespace are treated as if they have internal linkage**, which means that the content of an unnamed namespace can’t be seen outside of the file in which the unnamed namespace is defined.